### PR TITLE
Issue64 headless linux

### LINKED ISSUE #64: Add support for headless Linux
This ticket is to add support for headless Linux installation with no xdg (and no browser).
Currenlty, running `python -c "import buildingspy.development.regressiontest as t; t.Tester(tool=\"dymola\").report()"` yields the following errors.
```sh
/bin/sh: 1: xdg-settings: not found
argument of type 'NoneType' is not iterable
```
Instead, the server should run properly and its address should be printed in the terminal for remote access.
The necessary changes are ***backward compatible***.

### PR DESCRIPTION
**Pull request by AntoineGautier**
This closes #64.

### CODE REVIEW

File: pyfunnel/core.py [hunk 1]
```diff
@@ -407,10 +412,10 @@ def browse(self, *args, **kwargs):
                 while True:  # Prompt user to retry.
                     inp = input(
                         ('Launching browser yields syslog errors, '
-                         'probably because Chrome is used and the display entered screensaver mode.\n'
-                         'All related processes have been killed by precaution.\n'
-                         'If you have Firefox installed and want to use it persistently, enter p\n'
-                         'Otherwise, do you simply want to retry ([y]/n/p)? '))
+                            'probably because Chrome is used and the display entered screensaver mode.\n'
```
Comment (JayHuLBL) [inline pyfunnel/core.py:415]: Wonder if you want to has the extra spaces (or tabs)?

**Review by JayHuLBL — COMMENTED** [review]
@AntoineGautier Other than the little comment, the changes look good to me.

File: pyfunnel/core.py (hunk 1 above)
Comment (AntoineGautier) [reply pyfunnel/core.py:415]: I reformatted the string.

**Comment by JayHuLBL** [comment]
@AntoineGautier When using this development branch as the dependency of the BuildingsPy, running unit test for MBL models can successfully launch the browser and show the results difference if there is any.

It is ready to merge.

**Pull request merged**